crates/hive-utils/src/hive/genesis.rs

```diff
@@ -224,10 +225,10 @@ pub async fn serialize_hive_to_madara_genesis_config(
         madara_loader.storage.push(is_initialized);
     });
 
-    // Serialize the loader to a string
-    let madara_genesis_str = serde_json::to_string_pretty(&madara_loader)?;
-    // Write the string to a file
-    fs::write(combined_genesis, madara_genesis_str)?;
+    let combined_genesis_file =
+        File::options().create_new(true).write(true).append(true).open(combined_genesis_path).unwrap();
```

Comment:
we would want to overwrite this everytime we serialize so maybe `append(false)`

Reply:
Done [here](https://github.com/kkrt-labs/kakarot-rpc/pull/562/commits/f0a1c75e44c32990f28ea0f99131de8a69b641f0).